ramls/providers.raml

```diff
@@ -88,20 +89,36 @@ traits:
               example:
                 strict: false
                 value: !include examples/providers/providers_providerId_put_200_response.json
-  /packages:
-      get:
-        description: Search within a list of packages associated with a given provider.
-        is: [queriable, filterable, sortable, pageable]
-        responses:
-          200:
-            description: OK
-            body:
-              application/vnd.api+json:
-                description: List of providers matching the provider name and the total number of providers found.
-                type: packageCollection
-                example:
-                  strict: false
-                  value: !include examples/providers/providers_providerId_packages_get_200_response.json
+    /packages:
+        get:
+          description: Search within a list of packages associated with a given provider.
+          is: [queriable, filterable, sortable, pageable]
+          responses:
+            200:
+              description: OK
+              body:
+                application/vnd.api+json:
+                  description: List of providers matching the provider name and the total number of providers found.
```

Comment:
Minor adjustment - Should this description be something more like
`List of packages associated with a given provider matching search criteria`

Reply:
Thanks, updated